src/App.jsx

```diff
@@ -14,70 +16,77 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
-
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+export const App = () => {
+  const [value, setValue] = useState('Jam');
+  const [isCleared, setCleared] = useState(false);
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+  const selectValue = (good) => {
+    if (good !== value) {
+      setValue(good);
+      setCleared(false);
+    } else {
+      setValue(null);
+      setCleared(true);
+    }
+  };
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+  return (
+    <main className="section container">
+      {
+        isCleared
+          ? (
+            <h1 className="title is-flex is-align-items-center">
+              No goods selected
+            </h1>
+          )
+          : (
+            <h1 className="title is-flex is-align-items-center">
+              {`${value} is selected`}
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+              <button
+                data-cy="ClearButton"
+                type="button"
+                className="delete ml-3"
+                onClick={() => {
+                  setCleared(true);
+                  setValue(null);
+                }}
+              />
+            </h1>
+          )
+      }
 
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+      <table className="table">
+        <tbody>
+          {
+            goods.map(good => (
+              <tr
+                key={good}
+                data-cy="Good"
+                className={good === value ? 'has-background-success-light' : ''}
```

Comment:
I'd recommend learning to use the [classnames package](https://www.npmjs.com/package/classnames).